GitUI/CommandsDialogs/BrowseDialog/GitWorkingDirectoryStatusEventArgs.cs

```diff
@@ -14,9 +15,12 @@ public GitWorkingDirectoryStatusEventArgs(IEnumerable<GitItemStatus> itemStatuse
 
         public GitWorkingDirectoryStatusEventArgs()
         {
-            ItemStatuses = Enumerable.Empty<GitItemStatus>();
         }
 
+        /// <summary>
+        /// the statuses of all modified files/submodules or null if the previous information has got invalid
```

Comment:
suggestion: the status of all modified files/submodules or null if the previous information is invalidated